packages/notebook-scheduler/src/SubmitNotebook.ts

```diff
@@ -210,6 +214,8 @@ export class SubmitNotebook extends Widget
       imageSelect +
       '</td>' +
       '</tr>' +
+      tr +
+      td +
```

Comment:
not sure why you moved these, if the section below was uncommented now it wouldn't render correct